src/components/HTMLEngineProvider/HTMLRenderers/MentionUserRenderer.js

```diff
@@ -37,15 +39,28 @@ function MentionUserRenderer(props) {
     let accountID;
     let displayNameOrLogin;
     let navigationRoute;
+    const tnode = cloneDeep(props.tnode);
+    const getMentionDisplayText = (displayText, accountId, userLogin = '') => {
+        if (accountId && userLogin !== displayText) {
```

Comment:
we use capitals for other usages of `accountID`, so let's update this for consistency 
```suggestion
        if (accountID && userLogin !== displayText) {
```